### ISSUE TITLE
specify barcode resolution

### ISSUE THREAD

**Issue by helmerj** [anujluthra/barcode-generator#1]
Most barcode printers print at 300dpi to allow for a decent quality that barcode readers need in case of small barcode labels. 

Would it be possible to have a resolution option to specify the resolution for the generated barcodes?

Love the plug-in otherwise!

Cheers J.

**Comment by helmerj** [comment]
I actually implemented this myself, using the imagemagick_wrapper and additional option, I made theresolution and antialias option configurable in the barcode call. 

resolution = density [dpi]
antialias, boolean 1 => use antialiasing, 0 => do not use antialiasing

The new options are then passed to the imagemagick_wrapper.rb script which sets of the system call to convert. In case the new options are used they are inserted into an options-array which is then used to pass the options to convert in the system call.

PROBLEM:

The density options simply make the image larger so it would be the same to specify  larger with and height values. There also is a resample option for convert that I will experiment with. I would like to keep the image dimensions at higher res not simply make the image larger...

Cheers J.

patch: http://gist.github.com/333765
